webpack.config.mjs

```diff
@@ -0,0 +1,106 @@
+import path from 'path'
+import MiniCssExtractPlugin from 'mini-css-extract-plugin'
+import { VueLoaderPlugin } from 'vue-loader'
+import { BundleAnalyzerPlugin } from 'webpack-bundle-analyzer'
+import { fileURLToPath } from 'url'
+
+function resolve (dir) {
+    return path.join(path.dirname(fileURLToPath(import.meta.url)), dir.replace('./', ''))
+}
```

Comment:
for some reason git didn't manage to display the renaming and changes at once...

The `resolve` method changed